test_package/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.12)
+project(MiniGMPQRCodeTest)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+
+conan_basic_setup(KEEP_RPATHS)
```

Comment:
`conan_set_find_paths` would have less global side effects, but until we have the full guidelines for RPATHs and how to find the libraries during build this is fine.

Reply:
Yes we need to finalise all `RPATH` issues first.